src/main/java/Duke.java

```diff
@@ -1,10 +1,170 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int OFFSITE_OF_TIME = 4;
+    public static final int MAX_NUMBER_OF_TASKS = 100;
+
+    public static void greet(String logo){
+        System.out.println("    ____________________________________________________________");
```

Comment:
Consider making the __________________ a constant variable and just printing the variable